src/doc/unstable-book/src/library-features/enum-as-repr.md

```diff
@@ -0,0 +1,8 @@
+# `enum_as_repr`
+
+The tracking issue for this feature is FIXME
```

Comment:
To do before merging: Address this

Reply:
See above response: you file an issue using the library tracking issue template, then fill that in here.